api/src/main/java/org/pmiops/workbench/db/model/DbStorageEnums.java

```diff
@@ -207,6 +208,21 @@ public static Short dataAccessLevelToStorage(DataAccessLevel level) {
     return CLIENT_TO_STORAGE_DATA_ACCESS_LEVEL.get(level);
   }
 
+  // Cdr has Fitbit
+  private static final BiMap<CdrFitBitData, Short> CLIENT_TO_STORAGE_FITBIT =
+      ImmutableBiMap.<CdrFitBitData, Short>builder()
+          .put(CdrFitBitData.FALSE, (short) 0)
+          .put(CdrFitBitData.TRUE, (short) 1)
+          .build();
+
+  public static CdrFitBitData cdrFitBitDataFromStorage(Short level) {
+    return CLIENT_TO_STORAGE_FITBIT.inverse().get(level);
+  }
+
+  public static Short cdrFitBitDataToStorage(CdrFitBitData level) {
+    return CLIENT_TO_STORAGE_FITBIT.get(level);
+  }
+
```

Comment:
If we use type boolean, there is not need for these mthods.

Reply:
Done